eureka/S3_data_reduction/background.py

```diff
@@ -139,26 +152,13 @@ def fitbg(dataim, mask, x1, x2, deg=1, threshold=5, isrotate=False, isplots=Fals
                 dataslice = dataim[j,goodxvals]
                 # Check for at least 1 good x value
                 if len(goodxvals) == 0:
-                    #print(j,ny)
                     nobadpixels = True      #exit while loop
                     #Use coefficients from previous row
                 else:
                     # Fit along spatial direction with a polynomial of degree 'deg'
                     coeffs    = np.polyfit(goodxvals, dataslice, deg=degs[j])
                     # Evaluate model at goodexvals
                     model     = np.polyval(coeffs, goodxvals)
-                    #model = smooth.smooth(dataslice, window_len=window_len, window=windowtype)
-                    #model = sps.medfilt(dataslice, window_len)
-                    '''
-                    if isplots == 6:
```

Comment:
Rather than removing code for this figure, we should make sure it works for Eureka!

Reply:
How does this plot differ from the one on this line:
https://github.com/kevin218/Eureka/blob/a9f683bd7fc79d6fe3f2ba8debc334985ab18d1a/eureka/S3_data_reduction/background.py#L184

Do we want to be keeping both or just one?

Reply:
Ah ha, that'll do.  No need for both.